download.js

```diff
@@ -1,40 +1,59 @@
-const fs = require('fs')
-const publicKey = process.env.ONE_SKY_PUBLIC_KEY
-const privateKey = process.env.ONE_SKY_PRIVATE_KEY
-
-const onesky = require('onesky')(publicKey, privateKey)
+const { promises: fs } = require('fs')
+const oneSky = require('@brainly/onesky-utils')
 const sanitizeHtml = require('sanitize-html')
-const sanitizeOpts = {
-  allowedTags: ['b', 'strong', 'a', 'code'],
-  allowedAttributes: {
-    a: ['href', 'class']
-  },
-  textFilter(text) {
-    return text.replace(/\{\{/, '&#123;&#123;').replace(/\}\}/, '&#125;&#125;')
-  }
-}
```

Comment:
Can we keep this section untouched please to ease auditing/git-blame traversal?

Reply:
I think moving it to the bottom makes it generally more readable, as it put the implementation details up at the top, so I'd prefer to keep how it is. I think this is generally in-line with the policy of using hoisting to put exported functions at the top of a file (I don't think that's ever been written down though).

Reply:
> ever been written down

It may not be the best reference, but I've been pointing at https://github.com/overleaf/developer-manual/blob/9e3fe5ed6eb5b636829c5baba01f95f9092218de/code/creating-a-decaf-cleanup-PR.md#function-ifying

Reply:
I feel like this is falling into the constants category, which are hopefully still OK to keep at the top. Anyways let's move on.